code/datums/gamemode/factions/bloodcult/bloodcult.dm

```diff
@@ -91,6 +92,8 @@ var/veil_thickness = CULT_PROLOGUE
 	var/living_cultists = 0
 	var/living_noncultists = 0
 	for (var/mob/living/L in player_list)
+		if (issilicon(L))
```

Comment:
This still allows borers to slip through as living crew but it's an improvement

Reply:
Fixed.